### DISCUSSION TITLE
home assistant in docker, will this work?

### DISCUSSION THREAD

**Discussion by ecto1a2003** [mrlt8/docker-wyze-bridge#403] [Q&A]
My home assistant is running in docker, will this work with that? 

Also, does this make the cameras cloud free ?

**Comment by mrlt8** [comment]
Hmm, I think it should work as long as add-ons are supported in home assistant.

**Reply by ecto1a2003** [reply]
> Hmm, I think it should work as long as add-ons are supported in home assistant.

I clicked the get addon on and it seems that's only available for things with supervisor.

**Reply by mrlt8** [reply]
Could you run the bridge as a separate container? Most of the feature should work the same as the add-on.
